### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by aepsil0n** [npryce/adr-tools#94]
@npryce @npryce-springer are you still maintaining this tool? Do you want or need help with reviewing & merging pull requests?

I am noticing a steady influx of unaddressed PRs. I was packaging this script for NixOS. That made me curious whether there is enough momentum on this project to keep it well maintained.

**Comment by casidiablo** [comment]
Would be nice to have this in NixOS :+1:

**Comment by milibopp** [comment]
Here's the PR for that: https://github.com/NixOS/nixpkgs/pull/57901

I am beginning to consider a fork.

**Comment by jonathanjouty** [comment]
@edibopp do you use another tool as an alternative?

**Comment by pboling** [comment]
I was about to start using this tool.  Would love to see more momentum around it! 💯 I wish I had time to contribute to it, but my cup overfloweth already.

**Comment by milibopp** [comment]
@jonathanjouty no, I still have this tool installed, although I do not have an immediate need for ADRs at the moment. If there were at least one other person to help out, i would be willing to maintain a fork, as it is a nice tool and would be a shame to die off due to lack of maintenance – especially considering the tiny size of the codebase.

It is still some work to familiarize yourself with the code, go through all the past issues and pull requests, see what's relevant and take action. It can be done incrementally, of course. The next problem is discoverability. Most people likely find this tool through the 2011 blog post that points here, and forking may loose them.

I am still hopeful for @npryce-springer @npryce to show up at some point and look for a new maintainer or co-maintainer, if nothing else.

All that being said: adopting this tool is very low risk, as it is only a minor convenience tool over manually editing text files. You can easily adopt the ADR format without depending on the command line tool.

Another alternative would be to keep the format and rewrite the tool from scratch for fun ;)

**Comment by sata** [comment]
@edibopp I'm keen. I've introduced the usage of ADRs in one system that I maintain at work. 

I've pinged @npryce on Twitter as he seems active there.

**Comment by sata** [comment]
...no response, I'll be writing a adr-tools clone for our team needs.

**Comment by milibopp** [comment]
@sata do you intend to open-source it?

**Comment by sata** [comment]
@edibopp Yes, I'm hoping to have some time for it during Christmas holidays :)

**Comment by ms-ati** [comment]
@sata @edibopp Happy Holidays 2020! If either of you do get a maintained fork of `adr-tools` going, would you be so kind as to post a link to it here?

**Comment by keremciu** [comment]
heyya! I created a tool which doesn't require to be installed, you can run it wherever you have nodejs installed. https://github.com/keremciu/adr-tool

**Comment by ogborstad** [comment]
@sata how did your clone end up? dead or open soured? :)

**Comment by sata** [comment]
> @sata how did your clone end up? dead or open soured? :)

I would say unfortunately dead.. I've got something but it's nothing I'm ready to share... But thanks for reminding me, it's something I do want to work on!

It was easier to wrap it in a Docker image or having people installing it via brew/asdf than to write a new one.. Perhaps one day it will happen :)

**Comment by pboling** [comment]
@keremciu it looks great!  thanks

**Comment by keremciu** [comment]
you're welcome @pboling we use it at my company for long time, hopefully it can get better with more people adapting in their workflow and give feedback :) ✨

**Comment by chriscoffee** [comment]
I realise there's been little response here but would you like help maintaining this @npryce? I would be happy to help directly rather than just openly fork it.

**Comment by keremciu** [comment]
I can recommend trying my type-safe cli, it's super easy to try. I use it for long time and I welcome future ideas :)

```bash
npx adr-tool init
```

**Comment by XedinUnknown** [comment]
Bump. Still maintained?

**Comment by XedinUnknown** [comment]
About Docker: you can find an image that claims to have everything installed here. I had to override some values like `entrypoint` and `working_dir` in my `docker-compose.yml`, but otherwise looks like it's good.
https://github.com/rdhaliwal/adr-tools-docker
